frontend/src/metabase/query_builder/components/notebook/NotebookCell/NotebookCell.tsx

```diff
@@ -62,16 +66,21 @@ export const NotebookCellItem = forwardRef<
       data-testid={restProps["data-testid"] ?? "notebook-cell-item"}
       ref={ref}
     >
-      <NotebookCellItemContentContainer
-        inactive={inactive}
-        disabled={disabled}
-        readOnly={readOnly}
-        color={color}
-        roundedCorners={mainContentRoundedCorners}
-        style={containerStyle}
+      <Tooltip
+        label={t`${METAKEY}+click to open in new tab`}
```

Comment:
It looks like an incorrect abstraction. Notebook cells are not meant to be opened in the new tab. It's just that some consumers do. Please remove changes from here and add tooltip where we actually open the link.

Reply:
Good call. Thanks.
Moving it to the NotebookDataPicker simplified things a lot.

Please check the code again.